LICENSE.txt

```diff
@@ -4,342 +4,344 @@ can be downloaded from the following locations:
 	http://www.apache.org/licenses/LICENSE-2.0.txt
 
 This product also contains software under different licenses. This table below
-all the contained libraries (jar files) and the license under which they are 
+all the contained libraries (jar files) and the license under which they are
```

Comment:
Let's not change this file.This has to be generated.

Reply:
Fixed with [354ec26](https://github.com/wso2/product-micro-integrator/pull/3859/commits/354ec26c673fd6e54bf9c5f28b9d21abc00f6a45)